### PR TITLE
FAI-314: Score Cards: Move to kiegroup/kogito-tooling

### PR DESCRIPTION
**Pull request by manstis**
This is the PR following our discussions about moving to `kiegroup/kogito-tooling`....

### CODE REVIEW

**Comment by manstis** [comment]
The Full Downstream build seems to have errors in the Chrome Plugin integration tests.

This PR does not affect that area at all; so I suspect it's either transient or a known issue?

**Comment by tiagobento** [comment]
@manstis Yes.. @tomasdavidorg is already investigating it.

**Comment by manstis** [comment]
@tiagobento @ederign All green... all good to go?

**Comment by ederign** [comment]
@tiagobento can you please take a look at this?

**Comment by tiagobento** [comment]
@ederign @manstis Sorry about the delay on reviewing this PR, my tasks got longer than I anticipated. If you don't mind, I'll review it on Monday.

**Comment by manstis** [comment]
NP @tiagobento.

Don't forget this is not a code review.. things will be wrong; that's why we had the discussion about it.

The audit/check-list I shared with you (Google Docs) would need signing-off before this editor can be released.

There's lots of work to do before that!

**Review by tiagobento — APPROVED** [review]
Approving as we agreed per email. A more thorough review will be done before this package is publish on NPM or as part of one of our apps. Thank you @manstis! Looking forward to it!

**Comment by manstis** [comment]
Hi @sjamboro we just need your (notional) approval; then this can be merged...

File: packages/pmml-editor/src/__tests__/editor/components/LandingPage/molecules/ModelCard.test.tsx [hunk 1]
```diff
@@ -18,19 +18,22 @@ import * as React from "react";
...
-  baselineMethod: "max",
-  functionName: "regression"
-});
+// const model: Model = new Scorecard({
+//   modelName: "Name",
+//   Characteristics: { Characteristic: [] },
+//   MiningSchema: { MiningField: [] },
+//   baselineMethod: "max",
+//   functionName: "regression"
+// });
```
Comment (sjamboro) [inline packages/pmml-editor/src/__tests__/editor/components/LandingPage/molecules/ModelCard.test.tsx]: Please, remove commented code.

Comment (manstis) [reply packages/pmml-editor/src/__tests__/editor/components/LandingPage/molecules/ModelCard.test.tsx]: Done.

**Review by sjamboro — APPROVED** [review]

**Pull request merged**